utilities/utils.py

```
@@ -106,6 +108,76 @@ def _fetch_and_store_cacert(
+    Returns:
+        bool: True if MTV version >= min_version, False otherwise
+
+    Raises:
+        ValueError: If min_version or MTV version format is invalid
+        RuntimeError: If MTV operator not found or cluster client creation fails
+    """
+    if client is None:
+        try:
+            client = get_cluster_client()
+        except Exception as e:
+            LOGGER.error(f"Failed to create cluster client for MTV version check: {e}")
```

Ruff (0.14.11)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)